UPGRADE.md

```diff
@@ -487,6 +487,41 @@ export default (type, params) => {
 }
 ```
 
+## The `authProvider` Must Handle Permissions
+
+React-admin calls the `authProvider` to get the permissions for each page - using the `AUTH_GET_PERMISSIONS` verb. While in 2.x, implementing this `AUTH_GET_PERMISSIONS` verb was optional, it becomes compulsory in 3.0 as soon as you provide a custom `authProvider`. You can simply return a Promise resolved to an empty array to ignore permissions handling.
```

Comment:
I think you can return a resolved promise with nothing in it (`undefined`) as well. I'm afraid mentioning an empty array might confuse our users about what permissions are. Indeed, the result of this call may be a string, an object, an array, anything actually